doc/build/orm/basic_relationships.rst

```diff
@@ -674,23 +702,23 @@ class were available, we could also apply it afterwards::
     # first, module A, where Child has not been created yet,
     # we create a Parent class which knows nothing about Child
 
+
     class Parent(Base):
-        # ...
+        ...
```

Comment:
this is a bit of a change (remember these docs date back in some cases to Python 2.3).....but I guess this allows code formatters to work so OK